libraries/dynamic-ui-react/src/style/abstracts/variables/_dropdowns.scss

```
@@ -4,30 +4,29 @@
 
 // scss-docs-start dropdown-variables
 $dropdown-min-width: 10rem !default;
-$dropdown-padding-x: 0 !default;
+$dropdown-padding-x: .5rem !default;
 $dropdown-padding-y: .5rem !default;
 $dropdown-spacer: .125rem !default;
 $dropdown-font-size: $font-size-base !default;
 $dropdown-color: var(--#{$prefix}body-color) !default;
-$dropdown-bg: var(--#{$prefix}body-bg) !default;
+$dropdown-bg: var(--#{$prefix}white) !default;
 $dropdown-border-color: var(--#{$prefix}border-color-translucent) !default;
-$dropdown-border-radius: 0 !default;
+$dropdown-border-radius: .25rem !default;
 $dropdown-border-width: 0 !default;
 $dropdown-inner-border-radius: calc(#{$dropdown-border-radius} - #{$dropdown-border-width}) !default;
 $dropdown-divider-bg: $dropdown-border-color !default;
 $dropdown-divider-margin-y: var(--#{$prefix}ref-spacer-2) !default;
 $dropdown-box-shadow: 0 !default;
 
-$dropdown-link-color: var(--#{$prefix}body-color) !default;
-$dropdown-link-hover-color: $dropdown-link-color !default;
-$dropdown-link-hover-bg: var(--#{$prefix}tertiary-bg) !default;
+$dropdown-link-color: rgb(var(--#{$prefix}gray-500-rgb)) !default;
+$dropdown-link-hover-bg: rgb(var(--#{$prefix}gray-100-rgb)) !default;
```

```suggestion
$dropdown-link-hover-bg: rgb(var(--#{$prefix}gray-100-rgb)) !default;
$dropdown-link-hover-color: $dropdown-link-color !default;
```
si no le dejas un valor tomará la definición de bootstrap, que es la misma 

tratamos de dejar todas las variables que ofrece bootstrap escritas acá igual con !default; así no las cambiemos, nos deja ver rápidamente que variables tenemos disponibles.